python/src/aiconfig/editor/client/src/components/EditorContainer.tsx

```diff
@@ -81,6 +114,43 @@ export default function EditorContainer({
     [dispatch]
   );
 
+  const onAddPrompt = useCallback(
+    async (promptIndex: number, model: string) => {
+      const promptName = getDefaultNewPromptName(stateRef.current as AIConfig);
+      const newPrompt: Prompt = {
```

Comment:
Not a huge fan of this right now. Ideally we could just send up the model name and have the proper prompt structure inserted into the config to reconsolidate but I think we need to have the input schemas implemented in order to do so. Also, if no schema exists we will probably want to default to `{}` in the input with JSON renderer?

We can update as a follow-up

Reply:
Agreed with this